Index: lib/Catalyst/Authentication/Credential/OpenID.pm
===================================================================
--- lib/Catalyst/Authentication/Credential/OpenID.pm	(revision 11508)
+++ lib/Catalyst/Authentication/Credential/OpenID.pm	(working copy)
@@ -194,8 +194,8 @@
          <openid>
              <credential>
                  class   OpenID
+                 ua_class   LWP::UserAgent
              </credential>
-             ua_class   LWP::UserAgent
          </openid>
      </realms>
  </Plugin::Authentication>
@@ -208,7 +208,7 @@
      openid:
        credential:
          class: OpenID
-       ua_class: LWP::UserAgent
+         ua_class: LWP::UserAgent
 
 In a controller, perhaps C<Root::openid>-
 
@@ -353,24 +353,24 @@
                           }
               },
               openid => {
-                  consumer_secret => "Don't bother setting",
-                  ua_class => "LWP::UserAgent",
-                  ua_args => {
-                      whitelisted_hosts => [qw/ 127.0.0.1 localhost /],
-                  },
                   credential => {
                       class => "OpenID",
                       store => {
                           class => "OpenID",
                       },
-                  },
-                  extensions => [
-                      'http://openid.net/extensions/sreg/1.1',
-                      {
-                       required => 'email',
-                       optional => 'fullname,nickname,timezone',
+                      consumer_secret => "Don't bother setting",
+                      ua_class => "LWP::UserAgent",
+                      ua_args => {
+                          whitelisted_hosts => [qw/ 127.0.0.1 localhost /],
                       },
-                  ],
+                      extensions => [
+                          'http://openid.net/extensions/sreg/1.1',
+                          {
+                           required => 'email',
+                           optional => 'fullname,nickname,timezone',
+                          },
+                      ],
+                  },
               },
           },
       }
@@ -398,23 +398,23 @@
              </credential>
          </members>
          <openid>
-             <ua_args>
-                 whitelisted_hosts   127.0.0.1
-                 whitelisted_hosts   localhost
-             </ua_args>
-             consumer_secret   Don't bother setting
-             ua_class   LWP::UserAgent
              <credential>
                  <store>
                      class   OpenID
                  </store>
                  class   OpenID
+                 <ua_args>
+                     whitelisted_hosts   127.0.0.1
+                     whitelisted_hosts   localhost
+                 </ua_args>
+                 consumer_secret   Don't bother setting
+                 ua_class   LWP::UserAgent
+                 <extensions>
+                     http://openid.net/extensions/sreg/1.1
+                     required   email
+                     optional   fullname,nickname,timezone
+                 </extensions>
              </credential>
-             <extensions>
-                 http://openid.net/extensions/sreg/1.1
-                 required   email
-                 optional   fullname,nickname,timezone
-             </extensions>
          </openid>
      </realms>
  </Plugin::Authentication>
@@ -440,16 +440,16 @@
          class: OpenID
          store:
            class: OpenID
-       consumer_secret: Don't bother setting
-       ua_class: LWP::UserAgent
-       ua_args:
-         whitelisted_hosts:
-           - 127.0.0.1
-           - localhost
-       extensions:
-           - http://openid.net/extensions/sreg/1.1
-           - required: email
-             optional: fullname,nickname,timezone
+         consumer_secret: Don't bother setting
+         ua_class: LWP::UserAgent
+         ua_args:
+           whitelisted_hosts:
+             - 127.0.0.1
+             - localhost
+         extensions:
+             - http://openid.net/extensions/sreg/1.1
+             - required: email
+               optional: fullname,nickname,timezone
 
 B<NB>: There is no OpenID store yet.
 
@@ -459,10 +459,6 @@
 
 =head2 MORE ON CONFIGURATION
 
-These are set in your realm. See above.
-
-=over 4
-
 =item ua_args and ua_class
 
 L<LWPx::ParanoidAgent> is the default agent E<mdash> C<ua_class> E<mdash> if it's available, L<LWP::UserAgent> if not. You don't have to set
